include/tester/vtcp/tcp/TCPServer.hpp

```diff
@@ -15,20 +15,18 @@ using boost::asio::ip::tcp;
 class TCPServer
 {
 public:
-    TCPServer(const int port, std::shared_ptr<RequestHandler> requestHandler) :
+    TCPServer(const std::uint16_t port, std::shared_ptr<RequestHandler> requestHandler) :
         _listener(port, _context,
         std::bind(&TCPServer::onClientConnected, this, std::placeholders::_1, std::placeholders::_2)),
         _requestHandler(std::move(requestHandler))
-    {
-    }
+    {    }
```

Comment:
Put closing `}` on a new line (like you did previously, it was good 😄

Reply:
done